core/settings.py

```diff
@@ -158,7 +158,7 @@
 }
 
 SIMPLE_JWT = {
-    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=30),
+    'ACCESS_TOKEN_LIFETIME': timedelta(minutes=5),
```

Comment:
Reducing ACCESS_TOKEN_LIFETIME to 5 minutes may cause user experience issues due to frequent logouts. Consider increasing the token lifetime unless there is a clear security requirement.